framework/test/mocha/unit/modules/chain/forger.js

```diff
@@ -479,18 +481,19 @@ describe('forge', () => {
 
 				forgeModule.config.forging.delegates = [accountDetails];
 
-				forgeModule.loadDelegates(err => {
+				try {
+					await forgeModule.loadDelegates();
+				} catch (err) {
```

Comment:
This is giving false positive as it never catches and the expectation is never evaluated.

Try using the following instead:
```return expect(forgeModule.loadDelegates()).to.be.rejectedWith(`Invalid encryptedPassphrase for publicKey: ${accountDetails.publicKey}. Public keys do not match`);```